webapp/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Because the test runner (jest) doesn't use webpack, so it needs to get those settings from somewhere. This was the best way I could come up with doing it.